Selenium.Axe/Selenium.Axe.Test/AxeBuilderTest.cs

```diff
@@ -73,10 +203,32 @@ public void ShouldExecuteAxeScript()
             result.Incomplete.Length.Should().Be(0);
             result.Passes.Length.Should().Be(0);
             result.Violations.Length.Should().Be(0);
+        }
+        private void SetupAxeInjectionCall()
+        {
+            webDriverMock
+          .Setup(d => d.FindElements(It.IsAny<By>()))
```

Comment:
There are a few formatting inconsistencies; can you run this through an autoformatter of some sort?